### PR TITLE
Fixes Hero Image Size on Contact Page (#1256)

### LINKED ISSUE #1256: About page hero image take too much (90%) of space in desktop.
**I'm submitting a ...**
- [x] bug report
- [ ] feature request

**Current behavior:**

About page hero image take too much (90%) of space in desktop.

**Expected behavior:**

Should have size of 25% or less

**Screenshots of the issue:**

screenshot from 2018-12-05 11-28-15

**Will work on the issue:**
Yes

### PR DESCRIPTION
**Pull request by niteshkumarniranjan**
Fixes #1256 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [ ] I have added necessary documentation (if appropriate)
- [x] Added Surge preview link

#### Changes proposed in this pull request:

Made image size to be 35% of screen height so that contents are better visible.

- https://pr-1259-fossasia-susper.surge.sh

screenshot from 2018-12-05 13-22-16
screenshot from 2018-12-05 13-22-01

### CODE REVIEW

File: src/app/contact/contact.component.html [hunk 1]
```diff
@@ -29,11 +29,11 @@
 
 <div class="image-banner col-xs-12 col-sm-12 col-md-12 col-lg-12">
 
-  <img src="../../assets/images/boat.png" class="img-responsive banner">
-  <br>
-
+  <div class="img">
```
Comment (pythongiant) [inline src/app/contact/contact.component.html]: run this through a linter rest LGTM

**Comment by niteshkumarniranjan** [comment]
@praveenojha33 the package-lock.json isn't in the .gitignore. and the image will be cutdown if you will resize it.

**Comment by niteshkumarniranjan** [comment]
BTW. I'm fixing it.

**Comment by niteshkumarniranjan** [comment]
@praveenojha33 Fixed it and made image position to see full image.

**Comment by praveenojha33** [comment]
@niteshkumarniranjan The image is trimmed from the top. Please do not trim it.

**Comment by niteshkumarniranjan** [comment]
@praveenojha33 If Image will not be trimmed then how is it possible to change height.

**Comment by AakashMallik** [comment]
> @praveenojha33 If Image will not be trimmed then how is it possible to change height.

Scale it. @niteshkumarniranjan

**Comment by praveenojha33** [comment]
Please scale it.

**Comment by mariobehling** [comment]
Closing due to inactivity.

**Pull request closed without merging**